### ISSUE TITLE
Request: Autosave

### ISSUE THREAD

**Issue by olegsatov** [FartyPants/Twinbook#1]
Great work with the extension! Thank you! 👍

And it would be even greater if it could save all the texts between sessions. I mean memory, system, prefix, generated text and so on. Currently it's to easy to lose all the work.

**Comment by FartyPants** [comment]
Yeah, was thinking about it so many times - you refresh browser and boom - text is gone.
